### DISCUSSION TITLE
Rilevazione virus

### DISCUSSION THREAD

**Discussion by Claugo** [brentvollebregt/auto-py-to-exe#450] [Q&A]
This question has already been addressed and from what I understand there is nothing that can be done.  The problem is not so much transforming it into the .exe computer and keeping it on your own, I trust the author and I'm sure it's a false positive, but if you run this file on other computers, they turn up their noses a bit and trust the their antivirus, and eventually deletes the program out of fear.  This auto-py-to-exe is one of the most beautiful and working transformation programs, it's a shame that it is this problem.  I have other programs compiled with the previous version and the antivirus does not detect them as infected.  I tried reinstalling the previous version but it doesn't load.  It would be very helpful to fix this problem.  A thousand thanks.  Congratulations anyway because this is a great and valid program

**Comment by Claugo** [comment]
Mi rispondo da solo. La causa del rilevamento del trojan è dovuto alla funzione --noconsolle e per aggirare il problema al suo posto mettere --hide-console "hide-early". In questo caso la consolle viene aperta ma rimane attiva sulla barra dei programmi, perciò non visibile a schermo. Credo sia un'ottima soluzione per poter condividere il file con altri computer senza dover incappare in un blocco del file da parte dell'antivirus

**Comment by brentvollebregt** [comment]
Virus detection is covered in the help post under ["My Antivirus Detected the exe as a Virus"](https://nitratine.net/blog/post/issues-when-using-auto-py-to-exe/#my-antivirus-detected-the-exe-as-a-virus)
